Resources/Prototypes/Recipes/Crafting/Graphs/tiles.yml

```diff
@@ -0,0 +1,34 @@
+- type: constructionGraph
+  id: tileSteel
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: steeltile
+          steps:
+            - material: Steel
+              icon:
+                sprite: Objects/Materials/Sheets/metal.rsi
+                state: steel
+              name: steel sheet
+              amount: 1
+    - node: steeltile
+      entity: FloorTileItemSteel
+
+- type: constructionGraph
+  id: tileWood
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: woodtile
+          steps:
+            # FUCK IT 
+            - material: Steel
```

Comment:
confusion

Reply:
wait can't we use the existing wood planks/tower cap logs for this

Reply:
hence the confusion

Reply:
Not sure if this is related but I think swepts materials PR broke wood. So material should now be `MaterialWoodPlank1` instead of just Wood.

Reply:
That's not how it works peptide